cylc/flow/network/scan.py

```diff
@@ -271,10 +273,24 @@ def get_scan_items_from_fs(
 
             # Choose only suites with .service and matching filter
             if active_only:
+                # Skip suites running with cylc version < 8 (these suites
+                # do not have PUBLISH_PORT field)
                 try:
-                    contact_data = load_contact_file(
-                        reg, owner)
-                except (SuiteServiceFileError, IOError, TypeError, ValueError):
+                    contact_data = load_contact_file(reg, owner)
+                except (SuiteServiceFileError, IOError, TypeError) as exc:
+                    LOG.debug(f"Error loading contact file: {exc}")
```

Comment:
Testing, no issues yet.

This line results in a fairly uninformative message:

```
2020-04-21T10:49:13+01:00 DEBUG - Error loading contact file:
	Couldn't load contact file
```

Perhaps :
```suggestion
                    LOG.debug(f"Error loading contact file for: {reg}")
```